api/src/cli/commands/users/create.ts

```diff
@@ -11,7 +11,9 @@ export default async function usersCreate({ email, password, role }: any) {
 		const schema = await schemaInspector.overview();
 		const service = new UsersService({ schema, knex: database });
 
-		const id = await service.create({ email, password, role, status: 'active' });
+		const emailCaseInsensitive = email.toLowerCase();
+
+		const id = await service.create({ emailCaseInsensitive, password, role, status: 'active' });
```

Comment:
Instead of making a new const, lets just keep it inline:

```suggestion
		const id = await service.create({ email: email.toLowerCase(), password, role, status: 'active' });
```